fields/class-acf-field-rrule.php

```diff
@@ -470,6 +470,7 @@ function render_field( $field ) {
 							'value' => $field['value'] ? $field['value']['end_type'] : null,
 	                        'class' => 'end-type-select',
 							'choices' => array(
+								'none'	=> __('No end date', 'acf-rrule'),
```

Comment:
Wouldn't "Never" be a better option?

Reply:
Yeah. That would be a good alternative. I'll update in my next push.